### DISCUSSION TITLE
Adding image in M3

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1593] [Q&A] [answered]
It's not very common, but this week in the speech on "DO YOUR BEST IN THE MINISTRY", the speaker asked to show the table he cites in the source material, however, when he adds it, it is not in the correct section, is there an option for that?

![image](https://user-images.githubusercontent.com/90984283/221745318-3d1dc29e-1718-4d25-ab9d-3c08fb8fbccf.png)

**Comment by mtdvlpr** [comment, marked as answer]
Hi @e-Feitosa, unfortunately not. This feature is bound by limits and I don't see an easy way to fix that. You could try restarting M3 and opening the meeting without fetching again. That changes the way the headings are calculated.

**Reply by e-Feitosa** [reply]
I understand, a suggestion if it is possible to apply in all weeks, would be to define, 01-xx - TREASURES (everything related to section 1); 02-xx - APPLY YOURSELF... (all related to section 2) and lastly, 03-00 - LIVING AS CHRISTIANS (all related to section 3). But as I always explain, I don't know how to scale the work that it can take to include these procedures since it is something sporadic and the way it is works well.

**Comment by sircharlo** [comment]
> It's not very common, but this week in the speech on "DO YOUR BEST IN THE MINISTRY", the speaker asked to show the table he cites in the source material, however, when he adds it, it is not in the correct section, is there an option for that?

Sorry, I'm just looking into this. During what part was this? Trying to find that media you're talking about.

**Reply by e-Feitosa** [reply]
It is the table (Questions Young People Ask—Answers That Work, volume 2 p. 307-308) quoted in the source material of the speech.

![image](https://user-images.githubusercontent.com/90984283/222427136-0ac9bfe0-8092-468f-9bf3-f50b2d0d9f00.png)

![image](https://user-images.githubusercontent.com/90984283/222427299-16f3fda3-4e30-417b-afb3-167b23520851.png)

**Reply by sircharlo** [reply]
Understood. 

@mtdvlpr What do you think of the suggestion provided by @e-Feitosa ?
A naming convention such as: 

- 01-xx-xx - Treasures
- 02-xx-xx - Apply
- 03-xx-xx - Living

This would facilitate users putting custom media in the section they desire, and as a bonus, the "fallback" methods described [here](https://github.com/sircharlo/meeting-media-manager/pull/1384#issuecomment-1422264636) would no longer be necessary, as the meeting map would be used to name files correctly initially, upon fetching; no need to calculate where the headers should be when displaying the media list, just use the prefixes.

What do you think?

**Reply by mtdvlpr** [reply]
@sircharlo, I like it! The prefix would go from `part-nr-[optional sub nr]` to `heading-part-nr`.

**Comment by mtdvlpr** [comment]
@e-Feitosa, your suggestion has been merged and will be in the next release! It's already available in the latest beta release :)
